src/nimib.nim

```diff
@@ -132,22 +137,9 @@ template nbRawHtml*(content: string) =
 template nbJsFromStringInit*(body: string): NbBlock =
   var result = NbBlock(command: "nbCodeToJs", code: body, context: newContext(searchDirs = @[], partials = nb.partials), output: "")
```

Comment:
should we change name of the partial from `nbCodeToJs` to `nbCodeFromJs`?
```suggestion
  var result = NbBlock(command: "nbCodeFromJs", code: body, context: newContext(searchDirs = @[], partials = nb.partials), output: "")
```

Reply:
Yes that a good idea. I've been afraid to do it because I would miss to change it somewhere :rofl:

Reply:
Shouldn't it be `"nbJsFromCode"` though? :sweat_smile: